### PR TITLE
Check new expected element <ol class="toc">

### PR DESCRIPTION
**Pull request by tripu**
NB: throws only a warning if not found, ie this rule does *not* fail.

cf. #267.

### CODE REVIEW

File: lib/l10n-en_GB.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ exports.messages = {
 ,   "headers.h2-status.bad-h2": "Incorrect status h2 header."
     // headers/h2-toc
 ,   'headers.h2-toc.missing':  'There is no table of contents after the status section, labeled with a <code>&lt;h2&gt;</code> element with content <em>&ldquo;Table of Contents&rdquo;</em>.'
+    // headers/ol-toc
+,   'headers.ol-toc.not-found':  'The TOC should be an <code>&lt;ol class="toc"&gt;</code> inside the main navigation element (<code>&lt;nav id="toc"&gt;</code>).'
```
Comment (deniak) [inline lib/l10n-en_GB.js]: For the keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", I suggest we use `<span class="rfc2119">keyword</span>` like it appears in the old pubrules.

File: lib/rules/headers/ol-toc.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * Check the presence of the actual TOC (<code>&lt;ol class="toc"&gt;</code>) inside the main navigation element.
+ */
+
+exports.name  = 'headers.ol-toc';
+
+exports.check = function (sr, done) {
+
+    var toc = sr.$('#toc ol.toc');
```
Comment (deniak) [inline lib/rules/headers/ol-toc.js]: Search for `nav#toc` or `div#toc`

Comment (tripu) [inline lib/rules/headers/ol-toc.js]: Well spotted!
Done.

File: lib/l10n-en_GB.js (hunk 1 above)
Comment (tripu) [inline lib/l10n-en_GB.js]: This I'll address separately in #293

**Comment by tripu** [comment]
Ready to merge, @deniak.

**Comment by deniak** [comment]
:+1:

**Pull request merged**